stroke.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import re
+import os
+import time
+
+def get_stroke(hanzi):
+  return '&nbsp;'.join(map(get_stroke_single, hanzi))
+
+def get_stroke_single(hanzi):
+  if not os.path.exists('content/'):
+    os.mkdir('content/')
+
+  hanzi_id = str(ord(hanzi))
+  gif_loc = 'content/' + hanzi_id + '.gif'
+
+  if not os.path.exists(gif_loc):
+    response = requests.get('https://www.mdbg.net/chinese/rsc/img/stroke_anim/{}.gif'.format(hanzi_id))
+
+    if not response.ok and response.content:
+      print(u"Unable to fetch stroke for {}, trying again...".format(hanzi))
```

Comment:
there appears to be no retry...?

Reply:
preach

Reply:
no, like, I think you wrote this string wrong...

Reply:
yeah, so youre preaching the truth

ill fix it